.github/workflows/reading.yml

```diff
@@ -0,0 +1,22 @@
+name: reading
+concurrency:
+  group: reading
+on:
+  schedule:
+    - cron: '* * * * *'
+  workflow_dispatch:
+jobs:
+  Notify:
+    uses: ./.github/workflows/notify.yml
+    with:
+      cache_key: reading
+      slack_channel_id: ${{ vars.READING_SLACK_CHANNEL_ID }}
+      group_name: ${{ vars.READING_GROUP_NAME }}
+    secrets:
+      SEARCH_PARAMS: ${{ vars.READING_SEARCH_PARAMS }}
```

Comment:
So we search by the params with group id in there and post to `reading` slack channel.